mention_manager.py

```diff
@@ -12,8 +12,8 @@ def __init__(self, db, cow_bot):
     self.student_no_matcher = re.compile("e?\d{6,7}")
     self.db = db
     self.cow_bot = cow_bot
-    self.mention_text = "Your alias <b>{}</b> has been mentioned in " + \
-            "newsgroup: <b>{}</b> with header: <b>{}</b> at line: {}."
+    self.mention_text = "Your alias *{}* has been mentioned in " + \
+            "newsgroup: *{}* with header: *{}* at line: {}."
 
   def sendMention(self, cid, alias, newsgroup, header, line_no):
```

Comment:
you should also update this method too, github doesn't want me to comment that far from modifications so putting it in here.

bot.sendMsg assumes non-escaped strings by default and will perform escaping, hence asterisks `*` you add will be escaped instead of making the text bold. you should pass `escaped=True`.

In addition to that you really need to escape the string, so please replace `html.escape`s with the `escape` from `markdownrenderer.py`. You should also escape the `.` at the end of the `mention_text`, as telegrams markdown flavor says so :D

Reply:
Thanks for the swift review! Sending another commit that integrates your feedback.